src/caret_analyze/record/frequency.py

```diff
@@ -27,7 +27,7 @@ def __init__(
         target_column: Optional[str] = None
     ) -> None:
         """
-        Constructor.
+        Set a constructor.
```

Comment:
Could you replace sentence as follows?

```suggestion
        Construct an instance.
```

Reply:
@hsgwa 
I modified it all.